components/webui/imports/api/search/server/publications.js

```diff
@@ -46,10 +46,13 @@ Meteor.publish(Meteor.settings.public.SearchResultsMetadataCollectionName, ({sea
  * @param {string} publicationName
  * @param {object} props
  * @param {string} props.searchJobId
+ * @param {boolean} props.isExpectingUpdates pass true to
+ * observe result changes with polling rather than op-log.
```

Comment:
In the latest conclusions, we no longer re-enable oplogs, right?

```suggestion
 * @param {boolean} props.isExpectingUpdates pass true to enable polling on the 
 * Mongo collection.
```